IssueTracker/DataAccess/Repository/IRepository/IIssueRepository.cs

```diff
@@ -11,4 +11,5 @@ public interface IIssueData
     Task DeleteAsync(int id);
     Task NextStatusOfIssueAsync(int id, int statusId);
     Task PreviousStatusOfIssueAsync(int id, int statusId);
+    Task<int> GetProjectId(int id);
```

Comment:
This method has no references. Is it needed?

Reply:
it will be